alerter/build.gradle

```diff
@@ -14,7 +14,7 @@ apply from: rootProject.file('quality.gradle')
 
 final String GROUP_ID = "com.tapadoo.android"
 
-final String VERSION = "5.1.3"
```

Comment:
Hey @rezaiyan, thanks for the PR! Looks good, however I think it might be best to move to version 6.0.0, as this is a breaking change for those using the setSoundEnabled method!

Reply:
Sure,
I agree but at first, I decided to create a new function, and next, I thought it's better to be easy to use and not confusing.
However, I update the version.